configure.ac

```diff
@@ -141,12 +141,19 @@ AC_DEFINE_UNQUOTED([VA_DRIVER_INIT_FUNC], [$VA_DRIVER_INIT_FUNC],
     [Define driver entry-point])
 
 dnl Check for VA/DRM API
-USE_X11="$enable_x11"
-if test "$USE_X11" = "yes"; then
-    PKG_CHECK_MODULES(LIBVA_X11_DEPS, [libva-x11],
-      [AC_DEFINE([HAVE_VA_X11], [1], [Defined to 1 if VA/X11 API is enabled])],
-      [USE_X11="no"])
+USE_X11="no"
+if test "$enable_x11" = "yes"; then
```

Comment:
@chivakker Libva is **still** responsible for the backend, regardless of the move of the test folder, deps are still there.  Let's focus on the issue here.  What is the problem that you are trying to solve specifically for this project.